.php_cs.cache

```diff
@@ -0,0 +1 @@
hpdoc_inline_tag":true,"phpdoc_no_access":true,"phpdoc_no_alias_tag":true,"phpdoc_no_empty_return":true,"phpdoc_no_package":true,"phpdoc_no_useless_inheritdoc":true,"phpdoc_return_self_reference":true,"phpdoc_scalar":true,"phpdoc_separation":true,"phpdoc_single_line_var_spacing":true,"phpdoc_summary":true,"phpdoc_to_comment":true,"phpdoc_trim":true,"phpdoc_types":true,"phpdoc_var_without_name":true,"protected_to_private":true,"return_type_declaration":true,"semicolon_after_instruction":true,"short_scalar_cast":true,"single_blank_line_before_namespace":true,"single_class_element_per_statement":true,"single_line_comment_style":{"comment_types":["hash"]},"single_quote":true,"space_after_semicolon":{"remove_in_empty_for_expressions":true},"standardize_increment":true,"standardize_not_equals":true,"ternary_operator_spaces":true,"trailing_comma_in_multiline_array":true,"trim_array_spaces":true,"unary_operator_spaces":true,"whitespace_after_comma_in_array":true,"yoda_style":true,"blank_line_after_namespace":true,"elseif":true,"function_declaration":true,"indentation_type":true,"line_ending":true,"lowercase_constants":true,"lowercase_keywords":true,"no_break_comment":true,"no_closing_tag":true,"no_spaces_after_function_name":true,"no_spaces_inside_parenthesis":true,"no_trailing_whitespace":true,"no_trailing_whitespace_in_comment":true,"single_blank_line_at_eof":true,"single_import_per_statement":true,"single_line_after_imports":true,"switch_case_semicolon_to_colon":true,"switch_case_space":true,"visibility_required":true,"encoding":true,"full_opening_tag":true,"dir_constant":true,"ereg_to_preg":true,"error_suppression":true,"fopen_flag_order":true,"fopen_flags":true,"function_to_constant":true,"implode_call":true,"is_null":true,"modernize_types_casting":true,"native_constant_invocation":{"fix_built_in":false,"include":["DIRECTORY_SEPARATOR","PHP_SAPI","PHP_VERSION_ID"],"scope":"namespaced"},"native_function_invocation":{"include":["@compiler_optimized"],"scope":"namespaced"},"no_alias_functions":true,"no_homoglyph_names":true,"non_printable_character":{"use_escape_sequences_in_strings":false},"php_unit_construct":true,"psr4":true,"self_accessor":true,"set_type_to_cast":true,"array_syntax":{"syntax":"short"},"combine_consecutive_issets":true,"combine_consecutive_unsets":true,"header_comment":{"header":"This file is part of the Sonata Project package.\n\n(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>\n\nFor the full copyright and license information, please view the LICENSE\nfile that was distributed with this source code."},"no_php4_constructor":true,"no_useless_else":true,"no_useless_return":true,"ordered_class_elements":true,"ordered_imports":true,"phpdoc_order":true,"pow_to_exponentiation":true,"php_unit_namespaced":{"target":"5.7"},"php_unit_dedicate_assert":{"target":"5.6"},"php_unit_expectation":{"target":"5.6"},"php_unit_mock":{"target":"5.5"},"php_unit_no_expectation_annotation":{"target":"4.3"}},"hashes":{"tests\/bootstrap.php":2473808814}}
```

Comment:
This file should be removed.
Fixed in https://github.com/sonata-project/form-extensions/pull/15